third-party/linshare/src/test/resources/conf/home/vendor/font-awesome/scss/_stacked.scss

```diff
@@ -0,0 +1,20 @@
+// Stacked Icons
+// -------------------------
+
+.#{$fa-css-prefix}-stack {
```

Comment:
We really have to include all the Linshare website resources in an Apache project?